src/shared/components/Header/components/Sidebar/Sidebar.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import classNames from 'classnames';
-import Button from '../../../../shared/components/Button/Button';
-import './Sidebar.scss';
-import cancelIcon from '../../../../shared/images/sidebar/cancel.svg';
-import homeIson from '../../../../shared/images/sidebar/home.svg';
-import starIson from '../../../../shared/images/sidebar/star.svg';
+import Button from '../../../Button/Button';
+import './styles/Sidebar.scss';
```

Comment:
Please move styles import to be the last import. It's not an issue, but good for readability.